content/applications/finance/fiscal_localizations/chile.rst

```diff
@@ -2,898 +2,998 @@
+.. image:: chile/partner-dte-status.png
    :alt: Message with the commercial acceptance from the customer.
+   :align: center
 
-
-Processed for Claimed invoices
+Processed for claimed invoices
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Once the invoice has been Accepted by the SII **it can not be cancelled in Odoo**. In case you get
-a Claim for your customer the correct way to proceed is with a Credit Note to either cancel the
-Invoice or correct it. Please refer to the :ref:`chile/credit-notes` section for more details.
+Once the invoice has been accepted by the :abbr:`SII (Servicio de Impuestos Internos)`, **it can not
+be cancelled in Odoo**. In case you get a claim for your customer, the correct way to proceed is
+with a credit note to either cancel the invoice or correct it. Please refer to the
+:ref:`chile/credit-notes` section for more details.
 
-.. image:: chile/Chile24.png
+.. image:: chile/accepted-invoice.png
+   :alt: Invoice Commercial status updated to claimed.
    :align: center
-   :alt: Invoice Comercial status updated to Claimed.
 
-Common Errors
+Common errors
 ~~~~~~~~~~~~~
 
-There are multiple reasons behind a rejection from the SII, but these are some of the common errors
-you might have and which is the related solution.
-
-- Error: ``RECHAZO- DTE Sin Comuna Origen.``
-
-  *Hint:* Make sure the Company Address is properly filled including the State and City.
-
-- Error en Monto: ``- IVA debe declararse.``
-
-  *Hint:* The invoice lines should include one VAT tax, make sure you add one on each invoice line.
-
-- Error: ``Rut No Autorizado a Firmar.``
-
-  *Hint:* The invoice lines should include one VAT tax, make sure you add one on each invoice line.
-
-- Error: ``Fecha/Número Resolucion Invalido RECHAZO- CAF Vencido : (Firma_DTE[AAAA-MM-DD] -
-  CAF[AAAA-MM-DD]) &gt; 6 meses.``
-
-  *Hint:* Try to add a new CAF related to this document as the one you're using is expired.
-
-- Error: ``Element '{http://www.sii.cl/SiiDte%7DRutReceptor': This element is not expected. Expected
-  is ( {http://www.sii.cl/SiiDte%7DRutEnvia ).``
-
-  *Hint:* Make sure the field Document Type and VAT are set either in the Customer and in the main
-  company.
-
-- GLOSA: ``Usuario sin permiso de envio.``
-
-  *Hint:* This error indicates that most likely, your company has not passed the `Certification
-  process <https://www.sii.cl/factura_electronica/factura_mercado/proceso_certificacion.htm>`_ in
-  the SII - Sistema de Facturación de Mercado. If this is the case, please contact your Account
-  Manager or Customer Support as this certification is not part of the the Odoo services, but we
-  can give you some alternatives.
+There are multiple reasons behind a rejection from the :abbr:`SII (Servicio de Impuestos Internos)`,
+but these are some of the common errors you might have and how to solve them:
+
+- | **Error:** `RECHAZO- DTE Sin Comuna Origen`
+  | **Hint:** Make sure the company address is properly filled including the state and city.
```

Comment:
out of curiosity, what's this formatting with the | here for, Sam? (:

Reply:
@Felicious  the `|` character [forces a line break](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks). So it can be helpful to use in a list, if you want to break a line in a bullet point :)